gc/base/standard/ConcurrentGCSATB.cpp

```diff
@@ -348,6 +348,13 @@ MM_ConcurrentGCSATB::setupForConcurrent(MM_EnvironmentBase *env)
 
 	setThreadsScanned(env);
 	_stats.switchExecutionMode(CONCURRENT_INIT_COMPLETE, CONCURRENT_TRACE_ONLY);
+
+	TRIGGER_J9HOOK_MM_PRIVATE_GLOBAL_GC_INCREMENT_START(
+		_extensions->privateHookInterface,
+		env->getOmrVMThread() ,
+		omrtime_hires_clock(),
+		J9HOOK_MM_PRIVATE_CONCURRENT_SATB_TOGGLED,
+		true);
```

Comment:
could this be next to restoreGlobalFragmentIndex and then together in a class local wrapper enableSATB?